lib/bulk/common.js

```diff
@@ -5,23 +5,26 @@ const MongoError = require('mongodb-core').MongoError;
 const util = require('util');
 const toError = require('../utils').toError;
 const handleCallback = require('../utils').handleCallback;
+const applyWriteConcern = require('../utils').applyWriteConcern;
+const shallowClone = require('../utils').shallowClone;
+const ObjectID = require('mongodb-core').BSON.ObjectID;
```

Comment:
We need to use `retrieveBSON` for this, in case users are using a different bson implementation.

Reply:
Hold up, ignore. I thought we were in `core` :/